README.md

```diff
@@ -25,6 +27,10 @@ config :pool_max, :validate => :number, :default => 50
 # Max number of concurrent connections to a single host. Defaults to 25
 config :pool_max_per_route, :validate => :number, :default => 25
 
+# Specify this if you want to keep the TCP connection open between requests
+# using HTTP Keep-Alive. Defaults to 0 (off)
+config :keepalive, :validate => :number, :default => 0
```

Comment:
@cheald Hello! Can you help clarify something for me? We're using Manticore (Woo!) for http in Logstash and are wanting to expose some common settings. One setting, `:keepalive`, is documented as a boolean-or-fixnum, but I can't see what behavior a fixnum has other than being truthy to Manticore - for example, the behavior of `:keepalive => 1` seems the same as `:keepalive => 100`. Can you help me understand what the fixnum value for keepalive is (or may in the future?) be used for?